README.md

```diff
@@ -0,0 +1,16 @@
+Durable Task Plugin
+===
+
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/durable-task)](https://plugins.jenkins.io/durable-task)
+[![GitHub release](https://img.shields.io/github/v/release/jenkinsci/durable-task-plugin?label=changelog)](https://github.com/jenkinsci/durable-task-plugin/releases/latest)
```

Comment:
This link isn't super useful unless we start using Release Drafter to add release notes to the release page.

I think you should add a link to CHANGELOG.md like this somewhere in the README:

```
[Changelog](https://github.com/jenkinsci/durable-task-plugin/CHANGELOG.md)
```